File: corelib/autotest/log_simple.c
#include "corelib.h"

int main(void) {
	log_str("first message\n");
	log_str("second message\n");
	log("third message", 13);
	log_str("[PASS] log + log_str simple test\n");

	while (true) {}
	return 0;
}

File: rusty-corelib/examples/log_simple.rs
#![no_std]
#![no_main]

use core::hint::spin_loop;

use rusty_corelib::{entry, logln};

fn run() -> ! {
    let _ = logln!("first message");
    let _ = logln!("second message");
    let _ = logln!("third message");
    let _ = logln!("[PASS] rust log simple test");

    loop {
        spin_loop();
    }
}

entry!(run);
